src/js/base/ScrollList.tsx

```diff
@@ -59,3 +60,39 @@ export const ScrollList = ({ page, documents, pageCount, onLoadNextPage, renderR
 };
 
 StyledScrollList.displayName = "ScrollList";
+
+type StreamlinedScrollListProps = {
+    fetchNextPage: (options?: FetchNextPageOptions) => Promise<InfiniteQueryObserverResult>;
+    isFetchingNextPage: boolean;
+    isLoading: boolean;
+    items: unknown[];
+    renderRow: (item: unknown) => void;
+};
+
+export const StreamlinedScrollList = ({
```

Comment:
* What is streamlined compared to `ScrollList`? Need a docstring.
* Use `function`.

Reply:
The key difference was that `StreamlinedScrollList` was designed around using `react-queries` infinite scroll queries. Ended up taking the approach of instead renaming `StreamlinedScrollList` to `ScrollList` and the original `ScrollList` to `LegacyScrollList` as usage of the older version should be phased out as more redux logic gets stripped out. (or as we move to pagination)